pip-tools/requirements.in

```diff
@@ -1,6 +1,9 @@
-Flask-Migrate<4
-Flask-SQLAlchemy<3
-sqlalchemy==1.4.22
+Flask-Migrate
+# updating flasksqlalchemy to 3+ hits this bug: https://github.com/pallets-eco/flask-sqlalchemy/issues/1268
+Flask-SQLAlchemy>3
+# flask-sqlalchemy<3 calls apis deprecated in flask 2.3
+flask
+sqlalchemy
 # pin acme/pyopenssl until https://github.com/certbot/certbot/issues/9828 is fixed
 acme<2
 pyopenssl<23.3.0
```

Comment:
Should/can we remove some of these version constraints? I added them during my last attempt to update the package dependencies

Reply:
I looked at the linked issue and it's still open, so I think we're still stuck on this one